### PR TITLE
Add PulseDrop Direct Codec

### PR DESCRIPTION
**Pull request by brentmellor**
* Added PulseDrop direct codec and unit tests.
  * Copied and modified from the HDD codec and tests.
* Updated HDD codec to reflect the new 2.1V minimum low voltage threshold config. 
  * The Ti ADC can't run below 2.0V, hence the updated limit.
  * v29.2.0 firmware will ignore requests to set the value lower.

@CarterCall1 The unit tests pass, but I recommend reviewing them just in case I did something stupid.

### CODE REVIEW

File: pulsedrop_direct/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "codecId": "0032",
+  "sourceUrl": "https://github.com/the-real-vutility/payload-codecs/pulsedrop_direct",
+  "mac": "d46352",
+  "version": "1.0.0",
```
Comment (CarterCall1) [inline pulsedrop_direct/metadata.json:5]: Do you mind doing a minor version bump for the HDD codec in the metadata and package files? Something like 1.0.1 should be fine

**Review by CarterCall1 — APPROVED** [review]

File: pulsedrop_direct/metadata.json (hunk 1 above)
Comment (brentmellor) [reply pulsedrop_direct/metadata.json:5]: Sounds good... I updated the version attribute to 1.0.1 in the "metadata.json" and "package.json" files. Let me know if there's anywhere else I've missed.

Comment (CarterCall1) [reply pulsedrop_direct/metadata.json:5]: Looks good!

Comment (brentmellor) [reply pulsedrop_direct/metadata.json:5]: Is there a way we can test this codec against a real PulseDrop? (like put into a temporary Grafana dashboard?)
I'd love to prove it out before merging if it isn't a big task.

**Pull request merged**